leakcanary-android/src/main/AndroidManifest.xml

```diff
@@ -49,6 +51,7 @@
     <activity
         android:theme="@style/leak_canary_Theme.Transparent"
         android:name=".internal.RequestStoragePermissionActivity"
+        android:process=":leakcanary"
```

Comment:
I initially was going to do that but it requires changes to the `isInLeakProcess` method. I'm sure it's not that difficult of a change to make though.